extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4580,6 +4639,13 @@ var globalRequire = require;
 								this.videoControllerWrapper.raiseTimeUpdate(this.getCurrentTime(), this.getDuration());
 								_startTimeUpdater();
 							}
+							// Non-linear ad rules or VMAP ads will not be started by _tryStartAd()
+							// because there'll be no AMC ad pod. We start them here after the time update event
+							// in order to prevent the progress bar from flashing
+							if (_usingAdRules && !ad.isLinear())
+							{
+								_startNonLinearAdRulesOverlay();
+							}
 							break;
 						case eventType.RESUMED:
 							if (this.videoControllerWrapper)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)